tools/verify_scala_suffixes.sh

```diff
@@ -156,7 +156,9 @@ echo "Checking Scala-dependent modules:"
 
 for module in $infected; do
     out=`find . -maxdepth 3 -name 'pom.xml' -not -path '*target*' -exec grep "${module}</artifactId>" "{}" \;`
-    if [[ "$out" == "" ]]; then
+   # skip check flink-csv moudle, flink-csv format depends on dataformat which in flink-table-runtime-blink_${scala.binary.version}
```

Comment:
We maybe need wait @wuchong to extract `BaseRow` to table-common.
And then we can put `FileSystemFormatFactory` to table-common too.
So we don't need let csv depends on table-runtime.

Reply:
+1 for waiting `BaseRow` move to `table-common` module, I'll update the PR after that.